src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.CachedDocumentSearch.cs

```diff
@@ -51,6 +52,37 @@ private static void ClearCachedData()
             return cachedIndexMap != null && stringTable != null;
         }
 
+        public async Task SearchCachedDocumentsAsync(
```

Comment:
this is jsut a move from teh main file so that all stuff related to 'cached document search' is in a single file.